members/templates/members/union_members.html

```diff
@@ -0,0 +1,79 @@
+{% extends 'members/base.html' %}
+
+{% block content %}
+  <div class="container">
+      <h1>Foreningen for {{union.name}}</h1>
+      <strong>Adresse: </strong>{{union.address}}
+      {% if union.statues %}
+      <br>
+      <a href="{{union.statues}}"><strong>Vedtægter</strong></a>
+      {% endif %}
+      <br>
+      <br>
+      <h4>Bestyrelse</h4>
+      <div class="row" style="border:2px solid black;">
+        <div class="col-sm-6 col-centered">
+          Formand: <a href="mailto:{{union.chairman_email}}">{{union.chairman}}</a><br>
+          Næstformand: <a href="mailto:{{union.second_chair_email}}">{{union.second_chair}}</a><br>
+          Kasserer: <a href="mailto:{{union.cashier_email}}">{{union.cashier}}</a><br>
+          Sekretær: <a href="mailto:{{union.secratary_email}}">{{union.secretary}}</a><br>
+          Foreningsmail: <a href="mailto:{{union.union_email}}">{{union.union_email}}</a><br>
+        </div>
+        <div class="col-sm-6 col-centered">
+          <strong>Menige medlemmer</strong>
+          <div class="container" style="border:2px solid black;">
+            {{union.boardMembers|linebreaks}}
+            <!--{% for boardMember in union.boardMembers %}
+              <a href="mailto:{{boardMember.email}}">{{boardMember.name}}</a><br>
+            {% endfor %}-->
+          </div>
+        </div>
+      </div>
+      <br>
```

Comment:
Der er lige nogle ting til templaten. 

* Der skal ikke være inline styles, så flyt det over i en `scss` fil istedet. 
* Vi kører så vidt muligt ["semantisk" html](https://www.lifewire.com/why-use-semantic-html-3468271), så de der `br` skal væk. I stedet for at bruge `<br>` til at lave en liste så brug en `<ul>` du styler. 
* Det er helt fint at du bruger bootstrap til layout, men det er ved at være lidt gammelsdags. Det "nye" (IE 11 understøtter det) er at bruge flexbox i css. Her er en [lyn guide](https://www.youtube.com/watch?v=K74l26pE4YA) og et knald godt [cheat sheet](https://css-tricks.com/snippets/css/a-guide-to-flexbox/). Men det er mest hvis du synes det kunne være sjovt at lære. Kig evt. på headeren den bruger det også.

Reply:
- Ok, det prøver jeg at finde ud af.
- Ah, fair, jeg er ikke så meget inde i hvad der er god praksis i html, så jeg gjorde bare hvad der virkede, især fordi det jo ikke som sådan skulle være en liste med bullet points, men bare noget tekst over hinanden,
- Ingen anelse hvad du mener med at bruge bootstrap, jeg kopierede bare nogenlunde hvad jeg kunne se blev gjort på nogle af de andre sider.